nilearn/datasets/struct.py

```diff
@@ -523,57 +524,44 @@ def fetch_surf_fsaverage5(data_dir=None, url=None, resume=True, verbose=1):
 
 
 def _fetch_surf_fsaverage5(data_dir=None, url=None, resume=True, verbose=1):
-    if url is None:
-        url = 'https://www.nitrc.org/frs/download.php/'
+    """Helper function to ship fsaverage5 surfaces and sulcal information
+    with Nilearn.
+
+    The source of the data is coming from nitrc based on this PR #1016.
+    Manually downloaded gzipped and shipped with this function.
+
+    Shipping is done with Nilearn based on issue #1705.
+    """
 
-    # Preliminary checks and declarations
     dataset_name = 'fsaverage5'
-    data_dir = _get_dataset_dir(dataset_name, data_dir=data_dir,
-                                verbose=verbose)
 
     # Dataset description
     fdescr = _get_dataset_descr(dataset_name)
 
     # Download fsaverage surfaces and sulcal information
-    surf_file = '%s.%s.gii'
-    surf_url = url + '%i/%s.%s.gii'
-    surf_nids = {'lh pial': 9344, 'rh pial': 9345,
-                 'lh infl': 9346, 'rh infl': 9347,
-                 'lh sulc': 9348, 'rh sulc': 9349}
+    surface_file = '%s.%s.gii.gz'
+    surface_path = os.path.join(FSAVERAGE5_PATH, surface_file)
 
     pials = []
     infls = []
     sulcs = []
     for hemi in [('lh', 'left'), ('rh', 'right')]:
```

Comment:
it seems you don't need the hemi[0], i.e. 'lh' and 'rh' anymore, it was used for urls when files were retrieved from nitrc
you can just have `for hemi in 'left', 'right'`: ...

Reply:
and maybe instead of filling lists as was done before it is simpler to directly fill the resulting dictionary eg
```
result = Bunch()
# ...
result['pial_{}'.format(hemi)] = surface_path % ('pial', hemi)
# ...
...

but this is only a detail, feel free to ignore